yarn-project/end-to-end/src/e2e_fees/failures.test.ts

```diff
@@ -172,31 +172,36 @@ describe('e2e_fees failures', () => {
 
     await bananaCoin.methods.mint_public(aliceAddress, PublicMintedAlicePublicBananas).send().wait();
 
-    await expect(
-      bananaCoin.methods
-        .mint_public(aliceAddress, 1n) // random operation
-        .send({
-          fee: {
-            gasSettings,
-            paymentMethod: new BuggedTeardownFeePaymentMethod(bananaCoin.address, bananaFPC.address, aliceWallet),
-          },
-        })
-        .wait(),
-    ).rejects.toThrow(/invalid nonce/);
+    const badGas = GasSettings.from({
+      gasLimits: gasSettings.gasLimits,
+      inclusionFee: gasSettings.inclusionFee,
+      maxFeesPerGas: gasSettings.maxFeesPerGas,
+      teardownGasLimits: Gas.empty(),
+    });
 
-    // node also drops
     await expect(
       bananaCoin.methods
         .mint_public(aliceAddress, 1n) // random operation
         .send({
-          skipPublicSimulation: true,
           fee: {
-            gasSettings,
+            gasSettings: badGas,
             paymentMethod: new BuggedTeardownFeePaymentMethod(bananaCoin.address, bananaFPC.address, aliceWallet),
           },
         })
         .wait(),
-    ).rejects.toThrow(/Transaction [0-9a-f]{64} was dropped\. Reason: Tx dropped by P2P node\./);
+    ).rejects.toThrow();
+
+    const receipt = await bananaCoin.methods
+      .mint_public(aliceAddress, 1n) // random operation
+      .send({
+        skipPublicSimulation: true,
+        fee: {
+          gasSettings: badGas,
+          paymentMethod: new BuggedTeardownFeePaymentMethod(bananaCoin.address, bananaFPC.address, aliceWallet),
```

Comment:
Shouldn't we use a valid payment method, if we're already specifying a badGas for teardown?